### PR TITLE
Blood isn't quenchy.

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Purpose of change
It seems there's no supporting evidence for blood being hydrating or quenchy.
Basically between the salt and iron present in blood, it requires more water to flush excesses of these substances from the body than is taken in by drinking blood.

#### Describe the solution
Remove the quench value.  There's no strong evidence of exactly where the balance lies, so not setting it either positive or negative, just neutral.

#### Describe alternatives you've considered
Model the vitamins and such, and have commensurate costs for eliminating them, seems overcomplicated.
It seems reasonable to (if not useful) to centerfuge the blood to extract plasma, which would be hydrating, but that's kind of meh.

### CODE REVIEW

File: data/json/items/comestibles/carnivore.json [hunk 1]
```diff
@@ -132,8 +132,7 @@
     "healthy": 0,
     "parasites": 0,
     "material": "blood",
-    "quench": 5,
-    "comestible_type": "DRINK",
+blood-nob    "comestible_type": "DRINK",
```
Comment (kevingranade) [inline data/json/items/comestibles/carnivore.json:135]:
```suggestion
      "comestible_type": "DRINK",
```

**Comment by John-Candlebury** [comment]
If blood really has no vitamins how are vampires supposed to get vitamin D?

**Comment by Night-Pryanik** [comment]
> If blood really has no vitamins how are vampires supposed to get vitamin D?

Multivitamins to the rescue!

**Comment by KittyTac** [comment]
> If blood really has no vitamins how are vampires supposed to get vitamin D?

They aren't, that's why vampires are stereotypically pale and sickly.

**Pull request merged**